Frontend/src/data/item/columns.js

```diff
@@ -1,7 +1,17 @@
 import { saveParseTimestampToString } from "../../utils/utils.js";
+import Database from "../../database/Database"
 
 const backgroundColor = async (customer) => customer.highlight;
 
+async function countRentals(item_id) {
+  const selectors = [
+    {
+      item_id: item_id,
```

Comment:
```suggestion
      item_id: item_id,
      type: "rental"
```
Just in case some other doc_type ever has the field "item_id".